pmm-app/src/pmm-settings/components/Advanced/SwitchRow.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { FC } from 'react';
+import { Switch, useTheme } from '@grafana/ui';
+import { getSettingsStyles } from 'pmm-settings/Settings.styles';
+import { LinkTooltip } from 'shared/components/Elements/LinkTooltip/LinkTooltip';
+import { getStyles } from './Advanced.styles';
+
+
+export interface SwitchRowProps {
```

Comment:
Please move types to separate file named SwitchRow.types.ts and do it for all interfaces